doc/develop/tools/clion.rst

```diff
@@ -3,6 +3,10 @@
 CLion
 #####
 
+> **Updated:** This guide describes how to set up, build, and debug Zephyr’s sample application in CLion, using the IDE’s CMake integration. This approach is no longer optimal.
+>
+> CLion now features [native Zephyr West integration](https://www.jetbrains.com/help/clion/zephyr.html), which provides an easier and more intuitive way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but it’s still valid if you prefer to use CMake.
+
```

Comment:
we use ReStructured Text, not markdown :)
you prob want something like the following (make sure to add the actual hyperling associated with "native Zephyr West integration" at the bottom of the file.

```suggestion
.. note::

   This guide describes how to set up, build, and debug Zephyr's sample application in CLion, using
   the IDE's CMake integration. This approach is no longer optimal.

   CLion now features `native Zephyr West integration`_ which provides an easier and more intuitive
   way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but is still
   valid if you prefer to use CMake.
```

Reply:
Oh, got it :) Thanks for the help @kartben!